pkgs/development/python-modules/backports-zoneinfo/default.nix

```diff
@@ -27,8 +40,14 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
-  # unfortunately /etc/zoneinfo doesn't exist in sandbox, and many tests fail
-  doCheck = false;
+  disabledTests = [
+    # AssertionError: 'AEDT' != 'AEST'
+    "test_folds_and_gaps"
+    # backports.zoneinfo._common.ZoneInfoNotFoundError: 'No time zone found with key Eurasia/Badzone'
+    "test_bad_keys"
```

Comment:
This one is harder to explain away, should be xfail, but isn't. Something is off here, but I don't see the need to debug this further.